RS Metrics Reference (#2420)
RedisLabs/redislabs-docs: content/rs/clusters/monitoring/_index.md

---
Title: Monitoring with metrics and alerts
linkTitle: Monitoring

> description: Use the metrics that measure the performance of your Redis Enterprise Software (RS) clusters, nodes, databases and shards to keep an eye on the performance of your databases.

Reviewer: ```suggestion
description: Use the metrics that measure the performance of your Redis Enterprise Software (RS) clusters, nodes, databases, and shards to track the performance of your databases.
```
